src/SlashCommands.js

```diff
@@ -42,6 +43,11 @@ var commands = {
         return reject("Usage: /nick <display_name>");
     },
 
+    tint: function(room_id, args) {
```

Comment:
(most of) the other commands have a comment saying what they do. wouldn't hurt here.